### PR TITLE
add support for notify_keyspace_event

### PR DESCRIPTION
**Pull request by gkorland**

### CODE REVIEW

File: src/raw.rs [hunk 1]
```diff
@@ -474,3 +474,10 @@ pub fn subscribe_to_server_event(
 ) -> Status {
     unsafe { RedisModule_SubscribeToServerEvent.unwrap()(ctx, event, callback).into() }
 }
+
+#[cfg(feature = "experimental-api")]
+pub fn notify_keyspace_event(ctx: *mut RedisModuleCtx, event_type: i32, event: &str, keyname: *mut RedisModuleString) {
```
Comment (oshadmi) [inline src/raw.rs:479]: For `event_type`, we can also consider using `NotifyEvent` struct in raw.rs in redismodule-rs?

**Review by oshadmi — APPROVED** [review]

File: src/raw.rs (hunk 1 above)
Comment (gavrie) [reply src/raw.rs:479]: @oshadmi Sounds like a good idea!

Comment (gkorland) [reply src/raw.rs:479]: I thought about it, but I think you should be able to send a union of values

Comment (gavrie) [reply src/raw.rs:479]: @gkorland That's not a problem! We use the [bitflags](https://docs.rs/bitflags/1.2.1/bitflags/) crate which support this out of the box (e.g. `NotifyEvent::HASH | NotifyEvent::ZSET`).

Comment (gkorland) [reply src/raw.rs]: fixed

**Review by gavrie — APPROVED** [review]

**Pull request merged**